### PR TITLE
Todo list crud

### PR DESCRIPTION
**Pull request by shubhambhoyar077**
### **To Do List**

**Implemented CRUD functionality as follow:-**

  

- [ ]   Removed all hardcoded items from the tasks array.
- [ ]     Created a new JavaScript file for the new functionality.
- [ ]     Implemented function for adding a new task (add a new element to the array).
- [ ]     Implemented function for deleting a task (remove an element from the array).
- [ ]     Implemented function for editing task descriptions.

- [ ] By default new tasks should have the property completed set to false and the property index set to the value of the new array length.

   

- [ ]  All changes to the To Do List will be saved in local storage.

### **Run Instruction**

- [ ] Run **npm run build** cmd. It will create dist folder and open index.html in any browser.
- [ ] Or run **npm start** cmd. It's open project in port 8080.

### CODE REVIEW

File: src/modules/toDoList.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  displayList() {
+    this.toDoList.forEach((element) => {
+      renderList(element);
+    });
+  }
+
+  addItem(description) {
+    const newItem = {
+      index: this.toDoList.length,
```
Comment (TedLivist) [inline src/modules/toDoList.js:18]: - [ ] The `index` of your tasks should be the length of the array. The first task should have an `index` property of 1, the second 2, the 10th task should have an `index` of 10, and so on... Kindly refactor for that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @shubhambhoyar077,

Good job so far!
✔️ Update of the index values after deletion
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/toDoList.js (hunk 1 above)
Comment (shubhambhoyar077) [reply src/modules/toDoList.js]: Done updated as per instruction now index will start from 1 instead of 0.

**Review by fobadara — APPROVED** [review]
Hi @shubhambhoyar077, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passsing
✅ README looks profesional

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**